content/flux/v0.x/data-types/basic/boolean.md

```diff
@@ -2,11 +2,131 @@
 title: Work with booleans
 list_title: Boolean
 description: >
-  ...
+  Learn how to work with boolean data types in Flux.
 menu:
   flux_0_x:
     name: Boolean
     parent: Basic types
 weight: 201
 flux/v0.x/tags: ["basic types", "data types"]
----
\ No newline at end of file
+related:
+  - /flux/v0.x/stdlib/universe/bool/
+  - /flux/v0.x/stdlib/universe/tobool/
+---
+
+A **boolean** type represents a truth value (`true` or `false`).
+
+**Type name**: `bool`
+
+###### On this page:
+- [Boolean syntax](#boolean-syntax)
+- [Convert data types to booleans](#convert-data-types-to-booleans)
+- [Negate boolean values](#negate-boolean-values)
+
+## Boolean syntax
+Boolean literals are represented by the following:
+
+```js
+true
+false
+```
+
+## Convert data types to booleans
+Use the [`bool()` function](/flux/v0.x/stdlib/universe/bool/) to convert
+the following [basic types](/flux/v0.x/data-types/basic/) to booleans:
+
+- **string**: value must be `"true"` or `"false"`.
+- **float**: value must be `0.0` (false) or `1.0` (true).
+- **int**: value must be `0` (false) or `1` (true).
+- **uint**: value must be `0` (false) or `1` (true).
+
+```js
+bool(v: "true")
+// Returns true
+
+bool(v: 0.0)
+// Returns false
+
+bool(v: uint(v: 1))
```

Comment:
could this also be `bool(v: 1)`? If not, we may want to expand/clarify on the bool() funtion page

Reply:
It can be, there is just no way to express a uint in raw syntax. I'll add an example for just straight ints.